Demo/Application/Base/Settings/Settings.bundle/Root.plist

```diff
@@ -20,6 +20,7 @@
 				<string>PayPal - Web Checkout</string>
 				<string>Credit Card Tokenization</string>
 				<string>Venmo - Custom Button</string>
+				<string>Venmo Payment Button</string>
```

Comment:
Do we want to call this "Payment Buttons" and also add the PayPal button here? iirc that's what we did on Android.